README.md

```diff
@@ -18,6 +18,10 @@ Edna is an open-source tool for data analysis aimed to help researchers with the
 
 There is nothing to build yet.
 
+## Tests
```

Comment:
can be squashed with previous commit.

feel free to ignore

Reply:
Please move it to `backend` because it's about backend tests. I recently moved some information about backend to `backend/README.md`.